lib/screen_controller.dart

```diff
@@ -235,6 +236,31 @@ class ScreenController {
         throw RuntimeError(
             "openPlaidLink action requires the plaid's link_token.");
       }
+    } else if (action is PhoneContactAction) {
+      GetIt.I<ContactManager>().getPhoneContacts((contacts) {
+        if (action.getOnSuccess(dataContext) != null) {
+          executeActionWithScope(
+            context,
+            scopeManager!,
+            action.getOnSuccess(dataContext)!,
+            event: EnsembleEvent(
+              action.initiator!,
+              data: {'contacts': contacts},
+            ),
+          );
+        }
+      }, (error) {
+        if (action.getOnError(dataContext) != null) {
+          executeActionWithScope(
+            context,
+            scopeManager!,
+            action.getOnError(dataContext)!,
+            event: EnsembleEvent(
+              action.initiator!,
```

Comment:
Include error details in event?

Reply:
Added errors in the data param of EnsembleEvent